src/classes/PobControllerExt.cls

```diff
@@ -1,4 +1,9 @@
-public with sharing class PobControllerExt {
+public class PobControllerExt {
```

Comment:
Class `PobControllerExt` has grown too large. Please refactor to reduce the number of methods.